UnitTestFrameworkPkg/GoogleTest.md

```diff
@@ -742,9 +786,17 @@ With those limitations in mind, that completes the mock function
 declarations, and now the mock function definitions for those declarations
 can be created.
 
-### FunctionMockLib Mocks - Definition
 
-The definition of mock functions using the FunctionMockLib macros are done
+### FunctionMockLib Mocks - Definition
+The definition of mock functions using FunctionMockLib is done
+differently for library and non-library function mocks. Library mock
+functions are built using their own separate INF file and other mock
+functions (of a protocol, for example) are built in the source file of each GoogleTest
+that consumes them. This allows for multiple instantiation.
+To minimize repeated code, this is done by creating a macro in the mock header file - detailed later in this section.
+
+#### FunctionMockLib Mocks - Definition for Libraries
+The definition of library mock functions using the FunctionMockLib macros are done
 in source files. The name of the source file is determined by the interface
 (such as a library or a protocol) that is being created for the mock functions.

```

Comment:
`protocol` is removed below, but still present here.  Should this use of `protocol` be replaced with global table library?

Reply:
replaced with `(such as a library)`